src/app/contact/components/info.tsx

```diff
@@ -0,0 +1,170 @@
+	DOCS_URL_ROOT,
+} from '@/global/urls';
+import { Button, Link, Typography, css, useTheme } from '@icgc-argo/uikit';
+import { Col } from 'react-grid-system';
+
+const Info = () => {
+	const theme = useTheme();
+	return (
+		<Col
+			md={12}
+			lg={6}
+			style={{
+				padding: '0px 47px 0px 47px',
+				backgroundColor: theme.colors.white,
+			}}
+		>
+			<Typography
+				variant="hero"
+				css={css({
+					margin: '28px 0',
+				})}
+			>
+				Contact
+			</Typography>
+			<Typography
+				variant="subtitle2"
+				css={css({
+					margin: '21px 0',
+				})}
+			>
+				You may find the answer to your question in the following common topics:
+			</Typography>
+
+			<div
+				css={css`
+					display: flex;
+					flex-direction: column;
+					justify-content: space-between;
+					row-gap: 40px;
+					margin-top: 10px;
+				`}
+			>
+				<div>
+					<Typography
+						css={css`
+							margin: 0;
+						`}
+						variant="sectionHeader"
+						color="secondary"
+						bold
+						as="div"
+					>
+						Accessing Controlled Data{' '}
+					</Typography>
+					You will need to{' '}
+					<Link target="_blank" href={DACO_ROOT}>
+						apply to ICGC DACO
+					</Link>{' '}
+					in order to access controlled data. Visit our documentation for assistance with{' '}
+					<Link target="_blank" href={DOCS_DATA_ACCESS_PAGE}>
+						applying for access to controlled data.
+					</Link>
+				</div>
+				<div>
+					<Typography
+						css={css`
+							margin: 0;
+						`}
+						variant="sectionHeader"
+						color="secondary"
+						bold
+						as="div"
+					>
+						Downloading Data
+					</Typography>
+					<ul>
+						<li>
+							<Link target="_blank" href={DOCS_DATA_DOWNLOAD_PAGE}>
+								How to download data
+							</Link>{' '}
+							using the API Token and score-client
+						</li>
+						<li>
+							<Link target="_self" href={DOCS_API_TOKEN_PAGE}>
+								User profile and API token{' '}
+							</Link>
+						</li>
+					</ul>
+				</div>
+				<div>
+					<Typography
+						css={css`
+							margin: 0;
+						`}
+						variant="subtitle2"
+						color="secondary"
+						bold
+						as="div"
+					>
+						Submitting Data
+					</Typography>
+					<ul>
+						<li>
+							<Link target="_blank" href={DOCS_SUBMISSION_OVERVIEW_PAGE}>
+								Get started:
+							</Link>{' '}
+							a quick guide to data submission
+						</li>
+						<li>
+							How to{' '}
+							<Link target="_blank" href={DOCS_REGISTERING_SAMPLES_PAGE}>
+								register samples
+							</Link>
+						</li>
+						<li>
+							How to{' '}
+							<Link target="_blank" href={DOCS_SUBMITTING_CLINICAL_DATA_PAGE}>
+								submit clinical data
+							</Link>
+						</li>
+						<li>
+							How to{' '}
+							<Link target="_blank" href={DOCS_SUBMITTING_MOLECULAR_DATA_PAGE}>
+								submit molecular data
+							</Link>
+						</li>
+					</ul>
+				</div>
+
+				<div>
+					<Button variant="secondary">
+						<Link underline={false} target="_blank" href={DOCS_URL_ROOT}>
```

Comment:
link inside a button? is this a nextjs thing?

Reply:
No it's a styling uikit thing
not great markup wise (button should be on click action vs a link which is navigational) but it's how we do it design wise